docs/SECURITY-HW-REC.md

```diff
@@ -0,0 +1,30 @@
+# Hardware Security Recommendation
+
+## TPM
+
+TPM plays a key role in providing security in EVE, from the measured boot and device health attestation to storing cryptographic keys. To achieve EVE security requirements make sure the hardware is equipped with a TPM chip that supports at least TPM 2.0 standard with SHA256 banks available for use.
+
+## Memory
+
+EVE is capable of running virtual machines and containers provided by the user. The code executed from the user-provided application or VMs can not be fully trusted, as such hosting hardware should use memory that contains mitigations against [Rowhammer](https://www.wikiwand.com/en/Row_hammer) style attacks.
+
+## Boot Security Technologies
+
+EVE is leveraging TPM for remote attestation and secret sealing, but it relies on the fact that parts of the boot process that is out of the scope of EVE, are not malicious and can't lie about the measurements. To this end it is recommended to enable technologies like [Intel® Hardware Shield](https://www.intel.com/content/www/us/en/architecture-and-technology/hardware-shield-vpro-platform-security-paper.html), for example, Intel® Boot Guard in Measured Boot mode, this enables to measures the initial boot block (IBB) into TPM.
+
+## BIOS and UEFI Security Best Practices
+
+Correctly securing the BIOS and UEFI is another crucial step in reducing the possible attacks from outside of EVE, to this end, it is recommended to follow best practices laid out by NIST's BIOS Protection Guidelines [[800-147](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-147.pdf), [800-147B](https://nvlpubs.nist.gov/nistpubs/specialpublications/nist.sp.800-147b.pdf)] and NSA's [UEFI Defensive Practices Guidance](https://www.nsa.gov/portals/75/documents/what-we-do/cybersecurity/professional-resources/ctr-uefi-defensive-practices-guidance.pdf).
+
+## Firmware
+
+Make sure the firmware's authenticity and integrity are checked before execution and during update. Update the firmwares to the latest available version and make sure the implementation has mechanisms like rollback protection in place to prevent flashing an older vulnerable version to the system. Updating firmwares via [UEFI capsules](https://github.com/fwupd/fwupd) it is considered and will be supported by EVE.
+
+## Intel Hyperthreading
+
+It is recommended to disable Hyperthreading if the system is handling sensitive information. This technology has a reputation for being flawed and paving the road for side-channel attacks and information disclosures.
```

Comment:
Makes sense to check whether existing users of EVE does this or whether they depend on the additional performance one might see with HT. Ping me for a pointer.